ppmclibs/tinycv.pm

```diff
@@ -75,16 +76,16 @@ sub search_ {
     $stopwatch->lap("**++ search__: get image") if $stopwatch;
 
     my $img = $self;
-    for my $a (@{$needle->{area}}) {
-        push @exclude, $a if $a->{type} eq 'exclude';
-        push @match,   $a if $a->{type} eq 'match';
-        push @ocr,     $a if $a->{type} eq 'ocr';
+    for my $area (@{$needle->{area}}) {
+        push @exclude, $area if $area->{type} eq 'exclude';
+        push @match,   $area if $area->{type} eq 'match';
+        push @ocr,     $area if $area->{type} eq 'ocr';
     }
 
     if (@exclude) {
         $img = $self->copy;
-        for my $a (@exclude) {
-            $img->replacerect($a->{xpos}, $a->{ypos}, $a->{width}, $a->{height});
+        for my $e (@exclude) {
```

Comment:
Please don't go from one bad variable name to the next. I'd rather call this `$exclude_area`.

Reply:
Short variable names are perfectly fine if the scope is very limited, which is the case here. The greater the scope, the longer the variable name is one of the rules i personally follow. I'm not opposed to using longer variable names everywhere, but then we should be sure to enforce the rule consistently.